src/taskOperations.js

```diff
@@ -0,0 +1,147 @@
+export default class TaskOperations {
+  constructor(index, description, completed) {
+    this.index = index;
+    this.description = description;
+    this.completed = completed;
+  }
+
+  static tasks = [];
+
+  static descriptionInput = document.getElementById('get-task');
+
+  static taskList = document.querySelector('#taskList');
+
+  static addTask = () => {
+    const Task = new TaskOperations(TaskOperations.tasks.length,
+      TaskOperations.descriptionInput.value, false);
+
+    if (Task.description !== '') {
+      TaskOperations.tasks.push(Task);
+      localStorage.setItem('datas', JSON.stringify(TaskOperations.tasks));
+      TaskOperations.descriptionInput.value = '';
+      TaskOperations.renderTasks(Task);
+    }
+  }
+
+  static deleteTask = (index) => {
+    const taskBlock = document.getElementById(index);
+    const taskHolder = document.querySelectorAll('.taskItem');
+    TaskOperations.tasks = TaskOperations.tasks.filter((item) => item.index !== index);
+    TaskOperations.tasks.forEach((task) => {
+      if (task.index > index) {
+        task.index -= 1;
+      }
+    });
+    localStorage.setItem('datas', JSON.stringify(TaskOperations.tasks));
+    TaskOperations.taskList.removeChild(taskBlock);
+
+    if (taskHolder.length === 0) {
+      TaskOperations.taskList.innerHTML = '<h2>No tasks yet</h2>';
+    } else {
+      taskHolder.forEach((task) => {
+        if (task.id > index) {
+          task.id -= 1;
+        }
+      });
+    }
+  }
+
+  static editTask = (index) => {
+    document.querySelectorAll('#taskList li span .editDescription').forEach((node) => {
+      node.ondblclick = function createEditInput() {
+        const val = this.innerHTML;
+        const input = document.createElement('input');
+        input.type = 'text';
+        input.className = 'editInputBox';
+        input.value = val;
+        this.innerHTML = '';
+        this.appendChild(input);
+        input.focus();
+        input.onblur = function setLocalStorageAgain() {
+          const val = this.value;
+          this.parentNode.innerHTML = val;
+          TaskOperations.tasks.forEach((task) => {
+            if (task.index === index) {
+              task.description = input.value;
+              localStorage.setItem('datas', JSON.stringify(TaskOperations.tasks));
+            }
+          });
+        };
+      };
+    });
+  }
+
+  // static completeTask = (index) => {
+  //   TaskOperations.tasks[index].completed = true;
+  //   localStorage.setItem('datas', JSON.stringify(TaskOperations.tasks));
+  // }
+
+  // static uncompleteTask = (index) => {
+  //   TaskOperations.tasks[index].completed = false;
+  //   localStorage.setItem('datas', JSON.stringify(TaskOperations.tasks));
+  // }
+
+  // static sortTask = () => {
+  //   TaskOperations.tasks.sort((a, b) => a.index - b.index);
+  //   localStorage.setItem('datas', JSON.stringify(TaskOperations.tasks));
+  // }
```

Comment:
- [ ] Please, make sure to delete all the commented code to try to keep your code clean.